wordpress-mobile/WordPress-Android
# Login rework: Prologue animations

Introduces Lottie animations to the prologue screen. 4 promo screens are included in a ViewPager and auto-cycle. Manual swiping is also enabled.

The final visual touches, messages and final texts will be done in a future PR.

To test:
* With the app logged out or data-cleaned, open the app
* Notice the animations playing and the ViewPager auto-advancing after a few loops of each animation is performed
